scenarios/reproducers/va-nfv-ovs-dpdk-sriov.yml

```diff
@@ -0,0 +1,156 @@
+cifmw_architecture_scenario: "ovs-dpdk-sriov"
+
+# HERE if you want to overload kustomization, you can uncomment this parameter
+# and push the data structure you want to apply.
+# cifmw_architecture_user_kustomize:
+#   stage_0:
+#     'network-values':
+#       data:
+#         starwars: Obiwan
+
+# HERE, if you want to stop the deployment loop at any stage, you can uncomment
+# the following parameter and update the value to match the stage you want to
+# reach. Known stages are:
+#   pre_kustomize_stage_INDEX
+#   pre_apply_stage_INDEX
+#   post_apply_stage_INDEX
+#
+# cifmw_deploy_architecture_stopper:
+
+# What about some tempest?
+cifmw_run_tests: true
+cifmw_run_tempest: true
+cifmw_run_test_role: test_operator
+cifmw_test_operator_timeout: 7200
+cifmw_test_operator_tempest_include_list: |
+  tempest.scenario.test_network_basic_ops.TestNetworkBasicOps
+
+# This will instruct libvirt_manager to create
+# the ansible-controller, consuming the networks created and
+# managed by devscripts.
+# Note that the "osp_trunk" network is the equivalent of the
+# "private_net" in the CI layout, and will hold the VLAN for
+# network isolation.
+# The "ocpbm" network is managed by devscripts as well, and
+# provides access to Internet. This will be the equivalent of the
+# "public network" as seen in CI.
+cifmw_use_libvirt: true
+cifmw_virtualbmc_daemon_port: 50881
+cifmw_use_uefi: >-
+  {{ (cifmw_repo_setup_os_release is defined
+      and cifmw_repo_setup_os_release == 'rhel') | bool }}
+cifmw_root_partition_id: >-
+  {{
+    (cifmw_repo_setup_os_release is defined and cifmw_repo_setup_os_release == 'rhel') |
+    ternary(4, 1)
+  }}
+cifmw_libvirt_manager_net_prefix_add: false
+cifmw_libvirt_manager_fixed_networks:
+  - ocpbm
+  - ocppr
+  - osp_external
+  - osp_trunk
+cifmw_libvirt_manager_configuration:
+  networks:
+    ocpbm: |
+      <network>
+        <name>ocpbm</name>
+        <forward mode='bridge' />
+        <bridge name='ocpbm' />
+      </network>
+    ocppr: |
+      <network>
+        <name>ocppr</name>
+        <forward mode='bridge' />
+        <bridge name='ocppr' />
+      </network>
+    osp_external: |
+      <network>
+        <name>osp_external</name>
+        <forward mode='bridge' />
+        <bridge name='osp_external' />
+      </network>
+    osp_trunk: |
+      <network>
+        <name>osp_trunk</name>
+        <forward mode='bridge' />
+        <bridge name='osp_trunk' />
+      </network>
+  vms:
+    controller:
+      uefi: "{{ cifmw_use_uefi }}"
+      root_part_id: "{{ cifmw_root_partition_id }}"
+      image_url: "{{ cifmw_discovered_image_url }}"
+      sha256_image_name: "{{ cifmw_discovered_hash }}"
+      image_local_dir: "{{ cifmw_basedir }}/images/"
+      disk_file_name: "base-os.qcow2"
+      disksize: 50
+      memory: 8
+      cpus: 4
+      nets:
+        - ocpbm
+        - osp_trunk
+      ip_address:
+        address: <ip_address>
+        gw: <gw>
+        dns: <dns>
```

Comment:
should we add a `# changeme` comment at least?

Reply:
or just remove them @rdiazcam, cause we are patching them DS.

Reply:
even better if we can :)

Reply:
done